lib/ios/TabBarItemIOS15Creator.m

```diff
@@ -0,0 +1,37 @@
+#import "TabBarItemIOS15Creator.h"
+
+@implementation TabBarItemIOS15Creator
+
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 150000
+
+- (UITabBarItem *)createTabBarItem:(UITabBarItem *)mergeItem {
+    UITabBarItem *tabBarItem = [super createTabBarItem:mergeItem];
+    tabBarItem.scrollEdgeAppearance =
+        mergeItem.scrollEdgeAppearance ?: [[UITabBarAppearance alloc] init];
+    return tabBarItem;
+}
```

Comment:
@yogevbd thanks for looking into this PR! Out of curiosity, don't we need to set the regular appearance as well here? I'm asking because in `lib/ios/BottomTabPresenterCreator.m` you have a strict if-else not calling the pre-iOS 15 tab item creator.

Reply:
It will be created in the super call [here](https://github.com/wix/react-native-navigation/pull/7270/files#diff-729e5adb5dc3698d4499a5552e0e48a708328abad784540717995920364208efR7). I know it's not straight forward, will improve this soon.

Reply:
Oh ok, I missed that sorry! Your refactoring makes much more sense now!